ergotree-interpreter/src/sigma_protocol.rs

```diff
@@ -35,7 +35,7 @@ use derive_more::From;
 use derive_more::TryInto;
 
 /** The message sent by a prover to its associated verifier as part of a sigma protocol interaction. */
-pub(crate) trait ProverMessage {
+pub trait ProverMessage {
```

Comment:
Are you sure `ProverMessage` should be public? I don't see where it's used outside the crate.

Reply:
OW, Yes I missed that